util/zenfs.cc

```diff
@@ -374,7 +382,12 @@ int zenfs_tool_backup() {
   IOStatus io_status;
   IOOptions opts;
   IODebugContext dbg;
-  std::unique_ptr<ZonedBlockDevice> zbd = zbd_open(FLAGS_zbd, true, true);
+  std::unique_ptr<ZonedBlockDevice> zbd;
+
+  status = zenfs_zbd_open(FLAGS_zbd, true, true, zbd);
```

Comment:
After calling `zenfs_zbd_open` sometimes only the status is checked, and sometimes the `zbd` pointer is additionally checked to not be `null`. The check should always include `!zbd` or not?

Reply:
It is proper defensive programming to check both values, I guess. But I think both are checked? Some locations zbd is checked further down in code that is not touched by this patch.